main/modules/files.js

```diff
@@ -434,19 +438,29 @@ function createFromScrivener(importedPath, sender, isLoggedIntoPro) {
   )
 
   if (isLoggedIntoPro) {
-    importedJsonPromise.then((importedJson) => {
-      sender.send('create-plottr-cloud-file', importedJson, storyName)
-    })
+    importedJsonPromise
+      .then((importedJson) => {
+        sender.send('create-plottr-cloud-file', importedJson, storyName)
+      })
+      .catch((error) => {
+        log.error(error)
+        sender.send('finish-creating-local-scrivener-imported-file')
```

Comment:
This line doesn't seem right: we ran into an error and simply stopped without letting the user know what happened?
We should probably add a second listener in `_index.js` to handle errors.